File: src/Hospital/PatientEvent/Dto/EventDto.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Hospital.PatientEvent.Dto
{
    public class EventDto
    {
        public int Id { get; set; }
        public string EventName { get; set; }
        public string EventTime { get; set; }
        public int EventId { get; set; }

        public EventDto()
        {
        }

        public EventDto(int id, string eventName, string eventTime,int eventId)
        {
            Id = id;
            EventName = eventName;
            EventTime = eventTime;
            EventId = eventId;
        }
    }
}

File: src/PatientApp/Frontend/PatientApp/src/app/event/event.dto.ts
export class EventDto{
    public Id: number;
    public EventName: string;
    public EventTime: string;
    public EventId: number;
    constructor(){
        this.Id = 0,
        this.EventName = "",
        this.EventTime = "";
        this.EventId = 0;
    }
}